src/components/shared/formItems/ProjectPayoutMods.tsx

```diff
@@ -405,6 +405,7 @@ export default function ProjectPayoutMods({
               defaultValue={form.getFieldValue('beneficiary')}
               formItemProps={{
                 label: 'Address',
+                // extra: 'The address that should receive the tokens.',
```

Comment:
is there a reason we're leaving this in + commented out, or should we get rid of it?

Reply:
My bad there @peripheralist I did make a commit removing these but forgot to push!